playwright/masonry/utils/selectors.mjs

```diff
@@ -1,10 +1,10 @@
 // @flow strict
 
 const selectors = {
-  afterGrid: '.afterGrid',
-  gridItem: '[data-grid-item], .Collection-Item',
```

Comment:
This class name isn't valid due to our hashing, but also isn't needed. There aren't other `data-grid-item`s outside of Masonry.